demo/demo.md

```diff
@@ -116,23 +116,23 @@ Or vice-versa. This example will output an h1 header element with the `header--8
 
 In some cases the margin between elements may need to be adjusted. To do this, the `auro-header` component supports adjusting either the `top`, `bottom` or `both` margins. This coupled with the `size` prop, developers have a [full range of options](https://auro.alaskaair.com/components/auro/header/api).
 
-The following example shows how you can update the `bottom` margin to use the `xs` size setting.
+The following example shows how you can update the `bottom` margin to use the `50` size setting.
 
 Also note the use of the  util_stackMarginNone--top  utility selector on the ol from [Web Core Style Sheets](https://alaskaairlines.github.io/WebCoreStyleSheets/#utility-layout-mixin-auro_spacing).
 
 <div class="exampleWrapper">
-  <auro-header level="2" display="500" margin="bottom" size="xs">Hello World!</auro-header>
+  <auro-header level="2" display="500" margin="bottom" size="50">Hello World!</auro-header>
```

Comment:
Wouldn't changing the values we support be a breaking change since anyone using the old sizes wouldn't work any more?

Reply:
The old sizes still work with any current configuration. Updating the demo values is the process of deprecation to promote the new values over the legacy.